miio/integrations/vacuum/dreame/dreamevacuum_miot.py

```diff
@@ -190,12 +256,115 @@ def volume(self) -> str:
     def voice_package(self) -> str:
         return self.data["voice_package"]
 
+    @property
+    def timezone(self) -> str:
+        return self.data["timezone"]
 
-class DreameVacuumMiot(MiotDevice):
-    """Interface for Vacuum 1C STYTJ01ZHM (dreame.vacuum.mc1808)"""
+    @property
+    def cleaning_time(self) -> str:
+        return self.data["cleaning_time"]
+
+    @property
+    def cleaning_area(self) -> str:
+        return self.data["cleaning_area"]
+
+    @property
+    def first_clean_time(self) -> str:
+        return self.data["first_clean_time"]
+
+    @property
+    def total_clean_time(self) -> str:
+        return self.data["total_clean_time"]
+
+    @property
+    def total_clean_times(self) -> str:
+        return self.data["total_clean_times"]
+
+    @property
+    def total_clean_area(self) -> str:
+        return self.data["total_clean_area"]
+
+    @property
+    def cleaning_mode(self):
+        cleaning_mode = self.data["cleaning_mode"]
+        if self.model == DREAME_1C:
+            try:
+                return CleaningModeDreame1C(cleaning_mode)
+            except ValueError:
+                _LOGGER.error(f"Unknown CleaningMode ({cleaning_mode})")
+                return CleaningModeDreame1C.Unknown
+        elif self.model in [DREAME_D9, DREAME_F9]:
+            try:
+                return CleaningModeDreameF9(cleaning_mode)
+            except ValueError:
+                _LOGGER.error(f"Unknown CleaningMode ({cleaning_mode})")
+                return CleaningModeDreameF9.Unknown
 
-    mapping = _MAPPING
-    _supported_models = ["dreame.vacuum.mc1808"]
+    @property
+    def life_sieve(self) -> Optional[str]:
+        try:
+            return self.data["life_sieve"]
+        except KeyError:
+            return None
```

Comment:
```suggestion
    return self.data.get("life_sieve")
```
It's better to simply use `get` if there is no specific reason to log a warning or do something else.